tests/data/placebo/test_rds_delete_copy_restore/rds.ListTagsForResource_2.json

```diff
@@ -1,52 +1,52 @@
 {
-    "status_code": 200, 
+    "status_code": 200,
```

Comment:
all of the ListTagsForResource in the test directory should be obsolete with this pr.